docs/content/concepts/assets/asset-materializations.mdx

```diff
@@ -40,32 +40,13 @@ One way of recording materialization events is to log <PyObject module="dagster"
 To make Dagster aware that we materialized an asset in our op, we can log an <PyObject module="dagster" object="AssetMaterialization" /> event using the method <PyObject object="OpExecutionContext" method="log_event" />. This would involve changing the following op:
 
 ```python file=/concepts/assets/materialization_ops.py startafter=start_materialization_ops_marker_0 endbefore=end_materialization_ops_marker_0
-from dagster import op
-
-
-@op
-def my_simple_op():
-    df = read_df()
-    remote_storage_path = persist_to_storage(df)
-    return remote_storage_path
+ENOENT: no such file or directory, open '/Users/smackesey/stm/code/elementl/oss/examples/docs_snippets/docs_snippets/concepts/assets/materialization_ops.py'
 ```
 
 into something like this:
 
 ```python file=/concepts/assets/materialization_ops.py startafter=start_materialization_ops_marker_1 endbefore=end_materialization_ops_marker_1
-from dagster import AssetMaterialization, op
-
-
-@op
-def my_materialization_op(context):
-    df = read_df()
-    remote_storage_path = persist_to_storage(df)
-    context.log_event(
-        AssetMaterialization(
-            asset_key="my_dataset", description="Persisted result to storage"
-        )
-    )
-    return remote_storage_path
+ENOENT: no such file or directory, open '/Users/smackesey/stm/code/elementl/oss/examples/docs_snippets/docs_snippets/concepts/assets/materialization_ops.py'
```

Comment:
Not sure what's going on here. Maybe a race condition with my change? Maybe related to the lint error you mentioned?